app/src/main/res/drawable/rectangle_background.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?><!-- https://stackoverflow.com/a/16161493/3689782 -->
```

Comment:
Not able to understand why do we need this, this file currently has no attributes.

Reply:
To bind a rectangle background resource added this file. I have updated shape to it. PTAL.